content/docs/reference/design-docs/conflict-detection/index.en.md

```diff
@@ -9,131 +9,210 @@ weight: 45
+ - spacing requirements are exclusive: simultaneous requirements for the same resource are conflicting
+ - zone and switch requirements are shareable: simultaneous requirements are satisfied if the resource configuration is identical
 
-The current model only considers the following two resources:
- - **zones**, which have one state per way to traverse it
- - **switches**, which have one state per position
+**For conflict detection to work, resource requirements have to be at least as extensive as what's required to guarantee that a train path will not be disturbed.**
 
-{{% pageinfo color="info" %}}
-Station platforms could also be resources, which would enable modeling platform use conflicts: if two tracks share a tiny platform, it may not be allowed to use both sides of the platform at once.
-{{% /pageinfo %}}
 
-### Requirements and conflicts
+## Routing conflicts
 
-A **requirement** describes what resource is needed by an actor, and when.
+### Context
 
-There are a few types of requirements:
- - **shareable**: the resource is required to be in a given configuration, for a given time lapse. Other shareable requirements with the same configuration can stack up.
- - **exclusive**: the resource is required to be in a given configuration, for a given actor, for a given time lapse. This requirement cannot be combined with others.
+For trains to proceed safely along their planned path:
 
-**Conflicts** are both real-life disturbances and the incompatible resource requirements which cause them. A disturbance is something that would cause a train to be unable to run as planned.
- - route setup delays due to unavailable resources
- - spacing conflicts due to a train catching up with another
+ - switches have to be moved in the appropriate position
+ - level crossings have to activate
+ - risks of collision with other trains have to be mitigated
 
-These real-life issues can be linked to conflicts, which arise from incompatible resource requirements:
- - an exclusive requirement overlaps another requirement
- - a shareable requirements overlaps an incompatible requirement
- - a resource does not have time to change state between non-overlapping, yet close, requirements
+In practice, the path of trains is partitioned into routes, which when set, ensure a train can safely follow the route.
 
-For example, there are **no** conflicts when:
- - a resource has non-overlapping exclusive requirements
- - a resource is shared by requirements which require the same configuration
+Routes have the following lifestyle:
 
-## Generating requirements
+ - As a train approaches the start of one of its routes, it is **called** by an operator. If all resources required to safely use the route are available, switches and level crossings start to move. If a resources is not available, say because another train has reserved a section of track, this process is delayed until all conditions are met.
```

Comment:
Nitpicky, feel free to ignore this. "say" sounds too familiar in this context in my opinion

```suggestion
 - As a train approaches the start of one of its routes, it is **called** by an operator. If all resources required to safely use the route are available, switches and level crossings start to move. If a resources is not available, e.g. because another train has reserved a section of track, this process is delayed until all conditions are met.
```